connector/mqtt/vernemq/k2v-bridge/app/config.js

```diff
@@ -1,21 +1,21 @@
-const hostname = process.env.HOSTNAME || 'k2v-bridge-verne';
-const toBoolean = (mode) => ((mode || false) && (mode.toString().toLowerCase().trim() === 'true' || Number(mode) > 0));
+const hostname = process.env.HOSTNAME || 'k2v-bridge';
+const unsecuredMode = (mode) => ((mode || false) && (mode.toString().toLowerCase().trim() === 'true' || Number(mode) > 0));
 
 const app = {
   logLevel: process.env.K2V_LOG_LEVEL || 'info',
-  baseDir: process.env.BASE_DIR || '/opt/k2v-verne',
+  baseDir: process.env.BASE_DIR || '/opt/k2v-bridge',
   hostname,
 };
 
 const mqtt = {
-  clientUsername: process.env.K2V_MQTT_USERNAME || 'k2v-bridge-verne',
+  clientUsername: process.env.K2V_MQTT_USERNAME || 'k2v-bridge',
   clientId: process.env.K2V_MQTT_CLIENT_ID || hostname,
   host: process.env.K2V_MQTT_HOST || 'vernemq-k8s',
-  port: parseInt(process.env.K2V_MQTT_PORT, 0) || 8883,
-  keepAlive: parseInt(process.env.K2V_MQTT_KEEPALIVE, 0) || 60,
-  secure: toBoolean(process.env.K2V_MQTT_SECURE),
+  port: parseInt(process.env.K2V_MQTT_PORT, 10) || 8883,
+  keepAlive: parseInt(process.env.K2V_MQTT_KEEPALIVE, 10) || 60,
+  secure: unsecuredMode(process.env.K2V_MQTT_SECURE || true),
```

Comment:
It's a little confusing! You are setting a `secure` parameter, but we call a function called `unsecuredMode` passing a secure parameter (process.env.K2V_MQTT_SECURE)`.
Shouldn't be better to use `parseSecureMode` instead of `unsecuredMode`?